README.md

```diff
@@ -4,24 +4,24 @@ Python courses for the scientific researcher
 This repository contains several courses for the benefit of scientific researchers,
 particularly in the fields of oceanography and meteorology.
 
-There are currently five courses:
+There are currently 5 courses:
 
 ### An introduction to numpy
-3.5 hours &mdash; depends on a basic Python background
-http://nbviewer.ipython.org/github/SciTools/courses/blob/master/course_content/notebooks/numpy_intro.ipynb?create=1
+* **3.5 hours** &mdash; depends on a basic Python background
+* [view the course noteboook](http://nbviewer.ipython.org/github/SciTools/courses/blob/master/course_content/notebooks/numpy_intro.ipynb?create=1)
 
 ### An introduction to matplotlib
-3 hours &mdash; depends on "An introduction to numpy"
-http://nbviewer.ipython.org/github/SciTools/courses/blob/master/course_content/notebooks/matplotlib_intro.ipynb?create=1
+* **3 hours** &mdash; depends on "An introduction to numpy"
+* [view the course notebook](http://nbviewer.ipython.org/github/SciTools/courses/blob/master/course_content/notebooks/matplotlib_intro.ipynb?create=1)
```

Comment:
Too little punctuation ⁉️ 

How about:
```
... depends on "an introduction to such-and-such." [View the course notebook](http://linky.com).
```

(I've been waiting years for an opportunity to interrobang.)